index.d.ts

```diff
@@ -0,0 +1,365 @@
+             * 
+             *  enforce('hello').isString(false) // falsy
+             */
+            isString(expect?: boolean): IEnforce & T;
+    
+            /**
+             * Checks if a value is of any type other than string
+             * 
+             * @example
+             *  enforce('hello').isNotString() // falsy
+             * 
+             *  enforce(['hello']).isNotString() // truthy
+             * 
+             *  enforce('hello').isNotString(false) // truthy
+             */
+            isNotString(expect?: boolean): IEnforce & T;
+    
+            /**
+             * Checks if a value is of type number
+             * 
+             * @example
+             *  enforce(143).isNumber() // truthy
+             *  
+             *  enforce(NaN).isNumber() // truthy! (NaN is of type 'number!')
+             * 
+             *  enforce(143).isNumber(false) // falsy
+             */
+            isNumber(expect?: boolean): IEnforce & T;
+    
+            /**
+             * Checks if a value is of any type other than number
+             * 
+             * @example
+             *  enforce(143).isNotNumber() // falsy
+             * 
+             *  enforce('143').isNotNumber() // truthy
+             */
+            isNotNumber(expect?: boolean): IEnforce & T;
+    
+            /**
+             * Checks if your enforce value is empty, false, zero, null or undefined
+             * 
+             * @example
+             *  enforce([]).isEmpty() // truthy
+             * 
+             *  enforce('').isEmpty() // truthy
+             * 
+             *  enforce({}).isEmpty() // truthy
+             */ 
+            isEmpty(expect?: boolean): IEnforce & T;
+    
+            /**
+             * Checks that your enforce value is not empty, false, or zero
+             * 
+             * @example
+             * 
+             *  enforce([1]).isNotEmpty() // truthy
+             * 
+             *  enforce({1:1}).isNotEmpty() // truthy
+             * 
+             *  enforce([]).isNotEmpty() // falsy
+             */
+            isNotEmpty(expect?: boolean): IEnforce & T;
+    
+            /**
+             * Checks that your enforce value is larger than a given number
+             * 
+             * @example
+             * 
+             *  enforce([1]).largerThan(0) // truthy
+             * 
+             *  enforce('').largerThan(0) // falsy
+             */
+            largerThan(size: number): IEnforce & T;
+    
+            /**
+             * Checks that your enforce value is larger than or equals a given number
+             * 
+             * @example
+             * 
+             *  enforce([1]).largerThanOrEquals(1) // truthy
+             * 
+             *  enforce(0).largerThanOrEquals(1) // falsy
+             */
+            largerThanOrEquals(size: number): IEnforce & T;
+    
+            /**
+             * Checks that your enforce value equals the size than a given number
```

Comment:
```suggestion
             * Checks that your enforce value equals the size of a given number
```
(I need to fix it in the docs as well)